cmake/scripts/SquirrelIncludes.cmake

```diff
@@ -17,10 +17,14 @@ set(ARGC 1)
 set(ARG_READ NO)
 
 # Read all the arguments given to CMake; we are looking for -- and everything
-# that follows. Those are our api files.
+# that follows, until ||. Those are our api files.
```

Comment:
Maybe leave some clues as to what `||` is about, if you happen to know? For future-us :D